src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Data/SqlLite/ApplicationDbContextModelSnapshot.cs

```diff
@@ -14,12 +14,11 @@ protected override void BuildModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
             modelBuilder
-                .HasAnnotation("ProductVersion", "2.2.0-preview1");
```

Comment:
@divega @bricelam Should we force this to 3.0.0 now? We're unlikely to need to build these again.

Reply:
Sounds like the right thing to do to me, but I defer to @bricelam.

Reply:
Where's HasColumnType?

Reply:
I ran again with our nightlies and got the column types. Good catch.

Reply:
Now we can update to 3.0.0